code/modules/reagents/chemistry/recipes.dm

```diff
@@ -4,6 +4,23 @@
 	var/list/results = new/list()
 	var/list/required_reagents = new/list()
 	var/list/required_catalysts = new/list()
+		//FermiChem vars:
+	var/OptimalTempMin 		= 0 // Lower area of bell curve for determining heat based rate reactions
+	var/OptimalTempMax		= 1000 // Upper end for above
+	var/ExplodeTemp			= 9999 //Temperature at which reaction explodes
```

Comment:
why don't you just null it out?

Reply:
iirc it explodes as soon as the reaction procs

Reply:
```DM
if(isnull(ExplodeTemp))
    return
```
😏